index.js

```diff
@@ -125,3 +131,48 @@ app.listen(port, function() {
     if (flags.get('dry-run'))
         console.log('Starting in DRY-RUN mode');
 });
+
+// In addition to listening for notifications from GitHub, we regularly poll the
+// set of PRs to keep WebKit exports synchronized with the upstream PR.
+function pullRequestPoller() {
+    waitFor(60 * 1000).then(function() {
+        github.get("/repos/:owner/:repo/pulls", {}).then(function (pull_requests) {
+            pull_requests.forEach(function(pull_request) {
+                if (!webkit.related(pull_request.title)) {
+                    return;
+                }
+                metadata(pull_request.number, pull_request.user.login, pull_request.title, pull_request.body).then(function(metadata) {
+                    console.log("Issue: " + metadata.issue);
+                    console.log("Title: " + metadata.title);
+                    console.log("Labels: " + metadata.labels);
+                    console.log("isWebKitVerified: " + metadata.isWebKitVerified);
+                    console.log("isMergeable: " + metadata.isMergeable);
+                    console.log("reviewedDownstream: " + metadata.reviewedDownstream);
+
+                    if (metadata.isWebKitVerified) {
+                        console.log("Webkit flags:");
+                        if (metadata.webkit.flags.reviewed) {
+                              console.log("  Reviewed");
+                        }
+                        if (metadata.webkit.flags.inCommit) {
+                              console.log("  inCommit");
+                        }
+                    }
+                    // TODO: This part of the code must be uncommmented to
```

Comment:
I'm will land the uncomment of this myself in a follow-up commit once I've validated the change locally.

Reply:
OK. Let me know if I could do some additional verification in this regard